include/stronk/unit_v2.h

```diff
@@ -70,26 +96,34 @@ using stronk_default_unit = unit<Tag,
 template<unit_like UnitT, typename UnderlyingT>
 using unit_value_t = typename UnitT::template value<UnderlyingT>;
 
+template<typename ScaleT, unit_like UnitT, typename UnderlyingT>
+using unit_scaled_value_t = typename UnitT::template scaled_t<ScaleT>::template value<UnderlyingT>;
```

Comment:
Did not check usages yet, but here we order scale before unit, which is unlike the unit definition itself where Tag (unit) comes before scale.

Reply:
This was to get something which reads nicer